core/dom/elementpath.js

```diff
@@ -57,6 +57,11 @@
 		// Backward compact.
 		root = root || startNode.getDocument().getBody();
 
+		// Assign root value if startNode is null (https://dev.ckeditor.com/ticket/17028).
```

Comment:
@Comandeer  I'm a little bit confused, according to #470, there should be permanent link to issue in track. And it is there, or do I missed something?
Or by new convention you understand reference to github issue?